analytics_data_api/v0/models.py

```diff
@@ -206,3 +208,22 @@ class Video(BaseVideo):
 
     class Meta(BaseVideo.Meta):
         db_table = 'video'
+
+
+if settings.ELASTICSEARCH_LEARNERS_HOST:
+    connections.connections.create_connection(hosts=[settings.ELASTICSEARCH_LEARNERS_HOST])
```

Comment:
I think we should avoid executing logic like this on import.

I feel like this should really go in a "one-time startup" function like the one described here: https://docs.djangoproject.com/en/1.8/ref/applications/#django.apps.AppConfig.ready

I know we hacked together a "startup" function for the LMS, but since then I think Django has added first class support for that type of pattern. I assume it's the method I linked to above, but am not really sure, maybe worth talking to someone who is familiar with the newer versions of Django.